--- a/dlls/win32u/d3dkmt.c	2026-06-12 16:41:20.000000000 -0300
+++ b/dlls/win32u/d3dkmt.c	2026-06-23 00:53:33.233075849 -0300
@@ -491,7 +491,7 @@ static void d3dkmt_init_vulkan(void)
     static const struct vulkan_instance_extensions extensions =
     {
         .has_VK_KHR_get_physical_device_properties2 = 1,
-        .has_VK_KHR_external_fence_capabilities = 1,
+        .has_VK_KHR_external_memory_capabilities = 1,
     };
 
     d3dkmt_vulkan_instance = vulkan_instance_create( &extensions );
@@ -707,7 +707,7 @@ NTSTATUS WINAPI NtGdiDdDDIQueryAdapterIn
         if (desc->PrivateDriverDataSize < sizeof(*value))
             return STATUS_INVALID_PARAMETER;
 
-        *value = KMT_DRIVERVERSION_WDDM_3_1;
+        *value = KMT_DRIVERVERSION_WDDM_1_3;
         return STATUS_SUCCESS;
     }
     default:
